### PR TITLE
[REF] connector_acp: improve the support for multi files

### PR DESCRIPTION
**Pull request by dreispt**
Precursor to the Xerox document integration PR.
Supersedes part of #61

### CODE REVIEW

File: connector_acp/models/backend_acp.py [hunk 1]
```diff
@@ -16,9 +16,14 @@ class BackendAcp(models.Model):
     user = fields.Char(string="User",)
     password = fields.Char(string="Password",)
     active = fields.Boolean(string="Active", default=True)
-    status = fields.Selection((('unconfirmed', 'Unconfirmed'),
-                               ('confirmed', 'Confirmed')), string="Status",
-                              default='unconfirmed')
+    status = fields.Selection(
+        (("unconfirmed", "Unconfirmed"), ("confirmed", "Confirmed")),
+        string="Status",
+        default="unconfirmed",
```
Comment (max3903) [inline connector_acp/models/backend_acp.py:21]:
```suggestion
        default="unconfirmed",
        required=True,
```

File: connector_acp/models/backend_acp.py [hunk 2]
```diff
@@ -16,9 +16,14 @@ class BackendAcp(models.Model):
...
+    )
+    connection_type = fields.Selection(
+        [("nd", "Not defined")], string="Type of Connection", default="nd"
```
Comment (max3903) [inline connector_acp/models/backend_acp.py]:
```suggestion
        [("nd", "Not defined")], string="Type of Connection", default="nd",
        required=True
```

**Review by max3903 — APPROVED** [review]
Minor fixes. Not tested.

**Review by nelsonramirezs — APPROVED** [review]

**Pull request merged**